src/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+
+interface ResponseErrorInterface extends Error {
+    statusCode?: number,
+}
+
+interface HTTPResponseErrorInterface {
+    name: string,
+    statusCode: number,
+    message?: string
+}
+
+function errorHandlingMiddleware(error: ResponseErrorInterface, req: Request, res: Response, next: NextFunction): Response {
```

Comment:
Linter error for express error middleware : https://github.com/expressjs/generator/issues/78#issuecomment-103152636